{"level":"error","message":"Failed to connect to DB: TypeORMError: Entity metadata for Message#sender was not found. Check if you specified a correct entity object and if it's connected in the connection options.","project":"RealTimeChatApp","time":1696798550842,"timestamp":"2023-10-08T20:55:51.522Z"}
{"level":"error","message":"Failed to connect to DB: TypeORMError: Entity metadata for Message#sender was not found. Check if you specified a correct entity object and if it's connected in the connection options.","project":"RealTimeChatApp","time":1696798563241,"timestamp":"2023-10-08T20:56:03.887Z"}
{"level":"error","message":"Failed to connect to DB: TypeORMError: Entity metadata for Message#sender was not found. Check if you specified a correct entity object and if it's connected in the connection options.","project":"RealTimeChatApp","time":1696798573138,"timestamp":"2023-10-08T20:56:14.047Z"}
{"level":"error","message":"Failed to connect to DB: TypeORMError: Entity metadata for Message#sender was not found. Check if you specified a correct entity object and if it's connected in the connection options.","project":"RealTimeChatApp","time":1696798580433,"timestamp":"2023-10-08T20:56:21.394Z"}
{"level":"error","message":"Failed to connect to DB: TypeORMError: Entity metadata for Message#sender was not found. Check if you specified a correct entity object and if it's connected in the connection options.","project":"RealTimeChatApp","time":1696798587962,"timestamp":"2023-10-08T20:56:28.787Z"}
{"level":"error","message":"Failed to connect to DB: TypeORMError: Entity metadata for Message#sender was not found. Check if you specified a correct entity object and if it's connected in the connection options.","project":"RealTimeChatApp","time":1696798601453,"timestamp":"2023-10-08T20:56:42.155Z"}
{"level":"error","message":"Failed to connect to DB: TypeORMError: Entity metadata for Message#sender was not found. Check if you specified a correct entity object and if it's connected in the connection options.","project":"RealTimeChatApp","time":1696798608458,"timestamp":"2023-10-08T20:56:49.083Z"}
{"level":"error","message":"Failed to connect to DB: TypeORMError: Entity metadata for Message#sender was not found. Check if you specified a correct entity object and if it's connected in the connection options.","project":"RealTimeChatApp","time":1696798633310,"timestamp":"2023-10-08T20:57:13.875Z"}
{"level":"error","message":"Failed to connect to DB: TypeORMError: Entity metadata for Message#sender was not found. Check if you specified a correct entity object and if it's connected in the connection options.","project":"RealTimeChatApp","time":1696798640753,"timestamp":"2023-10-08T20:57:21.862Z"}
{"level":"error","message":"Failed to connect to DB: TypeORMError: Entity metadata for Message#sender was not found. Check if you specified a correct entity object and if it's connected in the connection options.","project":"RealTimeChatApp","time":1696798654334,"timestamp":"2023-10-08T20:57:34.884Z"}
{"level":"error","message":"Failed to connect to DB: TypeORMError: Entity metadata for Message#sender was not found. Check if you specified a correct entity object and if it's connected in the connection options.","project":"RealTimeChatApp","time":1696798663519,"timestamp":"2023-10-08T20:57:44.266Z"}
{"level":"error","message":"Failed to connect to DB: TypeORMError: Entity metadata for Message#sender was not found. Check if you specified a correct entity object and if it's connected in the connection options.","project":"RealTimeChatApp","time":1696798670351,"timestamp":"2023-10-08T20:57:51.002Z"}
{"level":"error","message":"Failed to connect to DB: TypeORMError: Entity metadata for Message#sender was not found. Check if you specified a correct entity object and if it's connected in the connection options.","project":"RealTimeChatApp","time":1696798683063,"timestamp":"2023-10-08T20:58:04.011Z"}
{"level":"error","message":"Failed to connect to DB: TypeORMError: Entity metadata for Message#sender was not found. Check if you specified a correct entity object and if it's connected in the connection options.","project":"RealTimeChatApp","time":1696798707202,"timestamp":"2023-10-08T20:58:28.002Z"}
{"level":"error","message":"Failed to connect to DB: TypeORMError: Entity metadata for Message#sender was not found. Check if you specified a correct entity object and if it's connected in the connection options.","project":"RealTimeChatApp","time":1696798714931,"timestamp":"2023-10-08T20:58:35.614Z"}
{"level":"error","message":"Failed to connect to DB: TypeORMError: Entity metadata for Message#sender was not found. Check if you specified a correct entity object and if it's connected in the connection options.","project":"RealTimeChatApp","time":1696798722445,"timestamp":"2023-10-08T20:58:42.961Z"}
{"level":"error","message":"Failed to connect to DB: TypeORMError: Entity metadata for Message#sender was not found. Check if you specified a correct entity object and if it's connected in the connection options.","project":"RealTimeChatApp","time":1696798730830,"timestamp":"2023-10-08T20:58:51.360Z"}
{"level":"error","message":"Failed to connect to DB: TypeORMError: Entity metadata for Message#sender was not found. Check if you specified a correct entity object and if it's connected in the connection options.","project":"RealTimeChatApp","time":1696798737485,"timestamp":"2023-10-08T20:58:58.156Z"}
{"level":"error","message":"Failed to connect to DB: TypeORMError: Entity metadata for Message#sender was not found. Check if you specified a correct entity object and if it's connected in the connection options.","project":"RealTimeChatApp","time":1696798745019,"timestamp":"2023-10-08T20:59:05.555Z"}
{"level":"error","message":"Failed to connect to DB: QueryFailedError: ER_TABLE_EXISTS_ERROR: Table 'user' already exists","project":"RealTimeChatApp","time":1696799090851,"timestamp":"2023-10-08T21:05:05.326Z"}
{"level":"error","message":"Failed to connect to DB: Error: ER_BAD_DB_ERROR: Unknown database 'realtimechatdb'","project":"RealTimeChatApp","time":1696843629724,"timestamp":"2023-10-09T09:27:10.184Z"}
{"level":"error","message":"Failed to connect to DB: QueryFailedError: ER_TABLE_EXISTS_ERROR: Table 'user' already exists","project":"RealTimeChatApp","time":1696845173864,"timestamp":"2023-10-09T09:53:00.082Z"}
{"level":"error","message":"Failed to connect to DB: QueryFailedError: ER_TABLE_EXISTS_ERROR: Table 'user' already exists","project":"RealTimeChatApp","time":1696845531697,"timestamp":"2023-10-09T09:58:57.224Z"}
{"level":"error","message":"Failed to connect to DB: QueryFailedError: ER_TABLE_EXISTS_ERROR: Table 'user' already exists","project":"RealTimeChatApp","time":1696845846870,"timestamp":"2023-10-09T10:04:12.925Z"}
{"level":"error","message":"Failed to connect to DB: QueryFailedError: ER_TABLE_EXISTS_ERROR: Table 'user' already exists","project":"RealTimeChatApp","time":1696846120720,"timestamp":"2023-10-09T10:08:46.039Z"}
{"level":"error","message":"Failed to connect to DB: QueryFailedError: ER_TABLE_EXISTS_ERROR: Table 'user' already exists","project":"RealTimeChatApp","time":1696846890865,"timestamp":"2023-10-09T10:21:37.297Z"}
{"level":"error","message":"Failed to connect to DB: QueryFailedError: ER_TABLE_EXISTS_ERROR: Table 'user' already exists","project":"RealTimeChatApp","time":1696847216893,"timestamp":"2023-10-09T10:27:03.092Z"}
{"level":"error","message":"Failed to connect to DB: QueryFailedError: ER_TABLE_EXISTS_ERROR: Table 'user' already exists","project":"RealTimeChatApp","time":1696851832349,"timestamp":"2023-10-09T11:43:58.895Z"}
{"level":"error","message":"Failed to connect to DB: QueryFailedError: ER_TABLE_EXISTS_ERROR: Table 'user' already exists","project":"RealTimeChatApp","time":1696865063983,"timestamp":"2023-10-09T15:24:43.316Z"}
{"level":"error","message":"Failed to connect to DB: QueryFailedError: ER_TABLE_EXISTS_ERROR: Table 'user' already exists","project":"RealTimeChatApp","time":1696865308968,"timestamp":"2023-10-09T15:28:42.216Z"}
{"level":"error","message":"Failed to connect to DB: QueryFailedError: ER_TABLE_EXISTS_ERROR: Table 'user' already exists","project":"RealTimeChatApp","time":1696867515825,"timestamp":"2023-10-09T16:05:33.433Z"}
{"level":"error","message":"Failed to connect to DB: QueryFailedError: ER_TABLE_EXISTS_ERROR: Table 'user' already exists","project":"RealTimeChatApp","time":1696868331089,"timestamp":"2023-10-09T16:19:05.589Z"}
{"level":"error","message":"Failed to connect to DB: QueryFailedError: ER_TABLE_EXISTS_ERROR: Table 'user' already exists","project":"RealTimeChatApp","time":1696868414166,"timestamp":"2023-10-09T16:20:31.444Z"}
{"level":"error","message":"Failed to connect to DB: QueryFailedError: ER_TABLE_EXISTS_ERROR: Table 'user' already exists","project":"RealTimeChatApp","time":1696868437256,"timestamp":"2023-10-09T16:20:59.280Z"}
{"level":"error","message":"Failed to connect to DB: QueryFailedError: ER_TABLE_EXISTS_ERROR: Table 'user' already exists","project":"RealTimeChatApp","time":1696868605234,"timestamp":"2023-10-09T16:23:38.986Z"}
{"level":"error","message":"Failed to connect to DB: QueryFailedError: ER_TABLE_EXISTS_ERROR: Table 'user' already exists","project":"RealTimeChatApp","time":1696868686503,"timestamp":"2023-10-09T16:25:06.126Z"}
{"level":"error","message":"Failed to connect to DB: QueryFailedError: ER_TABLE_EXISTS_ERROR: Table 'user' already exists","project":"RealTimeChatApp","time":1696868786187,"timestamp":"2023-10-09T16:26:39.377Z"}
{"level":"error","message":"Failed to connect to DB: QueryFailedError: ER_TABLE_EXISTS_ERROR: Table 'user' already exists","project":"RealTimeChatApp","time":1696868878216,"timestamp":"2023-10-09T16:28:16.528Z"}
{"level":"error","message":"Failed to connect to DB: QueryFailedError: ER_TABLE_EXISTS_ERROR: Table 'user' already exists","project":"RealTimeChatApp","time":1696869209295,"timestamp":"2023-10-09T16:33:43.077Z"}
{"level":"error","message":"Failed to connect to DB: QueryFailedError: ER_TABLE_EXISTS_ERROR: Table 'user' already exists","project":"RealTimeChatApp","time":1696874147537,"timestamp":"2023-10-09T17:56:09.654Z"}
{"level":"error","message":"Failed to connect to DB: QueryFailedError: ER_TABLE_EXISTS_ERROR: Table 'user' already exists","project":"RealTimeChatApp","time":1696882794281,"timestamp":"2023-10-09T20:20:12.578Z"}
{"level":"error","message":"Failed to connect to DB: QueryFailedError: ER_TABLE_EXISTS_ERROR: Table 'user' already exists","project":"RealTimeChatApp","time":1696884215565,"timestamp":"2023-10-09T20:43:51.587Z"}
{"level":"error","message":"Failed to connect to DB: QueryFailedError: ER_TABLE_EXISTS_ERROR: Table 'user' already exists","project":"RealTimeChatApp","time":1696884328845,"timestamp":"2023-10-09T20:45:43.038Z"}
{"level":"error","message":"Failed to connect to DB: QueryFailedError: ER_TABLE_EXISTS_ERROR: Table 'user' already exists","project":"RealTimeChatApp","time":1696884456307,"timestamp":"2023-10-09T20:47:50.221Z"}
{"level":"error","message":"Failed to connect to DB: QueryFailedError: ER_TABLE_EXISTS_ERROR: Table 'user' already exists","project":"RealTimeChatApp","time":1696884494146,"timestamp":"2023-10-09T20:48:29.909Z"}
{"level":"error","message":"Failed to connect to DB: QueryFailedError: ER_TABLE_EXISTS_ERROR: Table 'user' already exists","project":"RealTimeChatApp","time":1696884521603,"timestamp":"2023-10-09T20:48:54.906Z"}
{"level":"error","message":"Failed to connect to DB: QueryFailedError: ER_TABLE_EXISTS_ERROR: Table 'user' already exists","project":"RealTimeChatApp","time":1696884590736,"timestamp":"2023-10-09T20:50:05.496Z"}
{"level":"error","message":"Failed to connect to DB: QueryFailedError: ER_TABLE_EXISTS_ERROR: Table 'user' already exists","project":"RealTimeChatApp","time":1696884691703,"timestamp":"2023-10-09T20:51:47.224Z"}
{"level":"error","message":"Failed to connect to DB: QueryFailedError: ER_TABLE_EXISTS_ERROR: Table 'user' already exists","project":"RealTimeChatApp","time":1696885058161,"timestamp":"2023-10-09T20:57:52.501Z"}
